<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>441</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnHatirla">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçimi Hatırla</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbRenk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblRenk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arka Plan için Renk Seçiniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSecim">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçtiğiniz Renk :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDemirbas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDemirbasEkle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Demirbaş Ekle</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listDemirbas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSil">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydı Sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTemizle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Listeyi Temizle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTamam">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIptal">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
